.github/workflows/main.yml

```
@@ -0,0 +1,29 @@
+name: SpackBot CI
+
+on:
+  pull_request: []
+
+jobs:
+  formatting:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup black environment
+        run: |
+          conda create -c conda-forge -y -q --name black black
```

Feel free to merge this @adamjstewart if it's your preference (or update with what is)
```suggestion
          pip install black
```

---

I would personally use pip, but don't have a strong enough preference to make you change anything. Whatever works reliably is fine by me.